lib/payload.json

```
@@ -128,7 +128,11 @@
           "type": "object",
           "properties": {
             "speed": {
-              "description": "Wind speed (m/s)",
+              "description": "Instantaneous or average wind speed (m/s)",
+              "$ref": "#/definitions/speed"
+            },
+            "gustSpeed": {
+              "description": "Maximum wind speed during the measurement interval (m/s)",
```

I prefer not making assumptions on how the gust speed is calculated:
```suggestion
              "description": "Gust wind speed (m/s)",
```